src/services/adapters/registry.ts

```diff
@@ -8,7 +9,7 @@ import { ZeroAddress } from "../../helpers";
 import { structArray } from "../../struct";
 import { Address, Position, VaultDynamic, VaultStatic } from "../../types";
 
-export interface IRegistryAdapter {
+export interface RegistryAdapter {
```

Comment:
The rule that says that interfaces shouldn't start with an uppercase i

Reply:
hmm it's a good point, but the interface in Solidity is named like this, so not sure whether its best to keep it the same as the smart contract code, even though it's a bit of a code smell

Reply:
Should we put it back then?

Reply:
No I think let's leave as is